templates/success.html

```diff
@@ -11,17 +11,16 @@
 <body>
 
   <div class="container">
-    <h2>You are registered for the DESC July 2021 virtual meeting!</h2>
+    <h2>{{data.first_name}}, you are registered for the DESC October 2021 Sprint Week!</h2>
     <br>
     <br>
-    {{data.first_name}}, we are looking forward to seeing you at our virtual meeting!<br>
+    Whether in-person or remote, we are looking forward to sprinting together!<br>
```

Comment:
Maybe here we can say here something like: `If you have applied for in-presence participation, the hub you requested will contact you as soon as possible to confirm attendance.`

Reply:
The message now says: _Sprinters aiming to attend in person will receive an email as soon as possible from their intended hub to confirm participation and provide additional site-specific information._